src/window.rs

```diff
@@ -850,10 +854,15 @@ impl Default for CursorIcon {
     }
 }
 
+/// Fullscreen mode to be set on certain output.
+///
+/// Providing `None` to `Borderless` will result in fullscreen
+/// being set on a system preferred monitor. In general it'll be
+/// the current monitor.
```

Comment:
I'm not sure if I would say something about the "system preferred monitor" here, it could be easily confused with the primary monitor. Which is never used here afaics?

Reply:
Yeah, we can reword it to be current monitor?